src/components/BalancesList.js

```diff
@@ -514,6 +541,75 @@ function BalanceListItemDetails({ publicKey, serumMarkets, balanceInfo }) {
       : undefined
     : undefined;
 
+  const additionalInfo = isExtensionWidth ? undefined : (
+    <>
+      <Typography variant="body2" className={classes.address}>
+        Deposit Address: {publicKey.toBase58()}
+      </Typography>
+      <Typography variant="body2">
+        Token Name: {tokenName ?? 'Unknown'}
+      </Typography>
+      <Typography variant="body2">
+        Token Symbol: {tokenSymbol ?? 'Unknown'}
+      </Typography>
+      {mint ? (
+        <Typography variant="body2" className={classes.address}>
+          Token Address: {mint.toBase58()}
+        </Typography>
+      ) : null}
+      <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+        <div>
+          <Typography variant="body2">
+            <Link
+              href={
+                `https://explorer.solana.com/account/${publicKey.toBase58()}` +
+                urlSuffix
+              }
+              target="_blank"
+              rel="noopener"
+            >
+              View on Solana
+            </Link>
+          </Typography>
+          {market && (
+            <Typography variant="body2">
+              <Link
+                href={`https://dex.projectserum.com/#/market/${market}`}
+                target="_blank"
+                rel="noopener"
+              >
+                View on Serum
+              </Link>
+            </Typography>
+          )}
+          {swapInfo && swapInfo.coin.erc20Contract && (
+            <Typography variant="body2">
+              <Link
+                href={
+                  `https://etherscan.io/token/${swapInfo.coin.erc20Contract}` +
+                  urlSuffix
+                }
+                target="_blank"
+                rel="noopener"
+              >
+                View on Ethereum
+              </Link>
+            </Typography>
+          )}
+        </div>
+        {exportNeedsDisplay && wallet.allowsExport && (
```

Comment:
all of the things under "additionalInfo" don't appear in the extension window (including this), but when you click the "popout" button in the top left of the extension, they will show in expanded view.

Reply:
I see. We should add this functionality back into the extension later, then. Maybe we can combine a bunch of the toolbar items into a settings page, e.g. merge accounts, add token, export, edit name, all in one page.

Reply:
Hmm yeah. It might also not be necessary to add them to the extension window as long as they appear in the expanded view I think? Note that the expanded view does not open sollet.io; it opens the extension itself except in a full window with all of the features you've mentioned.

Reply:
If we do decide to leave them in the expanded view only, there should be some kind of signaling in the extension popup view that indicates that it exists in the expanded view, though.